packs/vsphere/actions/vmwarelib/inventory.py

```diff
@@ -25,30 +11,73 @@ def get_managed_entity(content, vimtype, moid=None, name=None):
         elif name and entity.name == name:
             return entity
 
+    #if this area is reached no object has been found
+    #if no arguments were passed just return with empty object
+    if ((name is None or name == "") and (moid is None or moid == "")):
```

Comment:
`if not name and not moid:` is more "pythonic". Checking for falsy does have its pitfalls.